packages/kit/src/runtime/server/index.js

```diff
@@ -257,6 +243,8 @@ export async function respond(request, options, state) {
 
 						return response;
 					}
+
+					break;
```

Comment:
this break at the very end confuses me a bit, is there a way to restructure the loop to avoid it? (or explain it to me like i'm 5)